org.eclipse.draw2d/src/org/eclipse/draw2d/graph/RankSorter.java

```diff
@@ -22,7 +23,7 @@
  */
 class RankSorter {
 
-	Random flipflop = new Random(3);
```

Comment:
SecureRandom can be significantly slower than the plain old Random. Do we really need a cryptographically strong generator here?

Reply:
By the looks of it, this change is redundant because randomization here is not used for any security purposes and possibly slows the overall performance of the sorter.

Reply:
Optimization cases which can risk in outages are also made in scope of understanding the data through the lenses of great mixture of AI and software engineering. That's the though worth taking to considerations, as sometimes he can be kinda silly and send either very serious stuff or very chill like that ;)

Reply:
I think only zip folder change can be viable here, could you update the pr to not include SecureRandom changes?